Gems/Atom/Feature/Common/Assets/Shaders/SkyAtmosphere/SkyAtmosphereCommon.azsli

```diff
@@ -523,64 +529,31 @@ SingleScatteringResult IntegrateScatteredLuminance(
         }
 
         // Earth shadow 
-        const float tEarth = RaySphereClosestHitWS(earthOrigin + PLANET_RADIUS_OFFSET_KM * upVector, Atmosphere.BottomRadius, P, sunDir);
+        const float tEarth = RaySphereClosestHitWS(earthOrigin + PLANET_RADIUS_OFFSET_KM * upVector, Atmosphere.BottomRadius, samplePos, sunDir);
         const float earthShadow = tEarth >= 0.0f ? 0.0f : 1.0f;
 
-        // Dual scattering for multi scattering 
```

Comment:
Multi scattering is removed?

Reply:
The multi-scattering functionality was never part of the initial commit - though we should add it!